### ISSUE TITLE
Accessibility issue: SWUbanner is missing alt text

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [pnpm/.github#2]
The StandWithUkraine image is missing alt text, which means users who cannot visually see the image won't know what's in it. Would you be open to me sending a PR that adds it in?

```diff
- [![SWUbanner](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md)
+ [![Ukrainian flag with text: Russia has invaded Ukraine and already killed tens of thousands of civilians, with many more raped or tortured. The death toll keeps climbing. It's a *genocide*. We need your help. Let's fight back against the Russian regime. Help Ukraine Now ->](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md)
```

[Explainer on using `alt` attributes](https://www.w3.org/TR/WCAG20-TECHS/H37.html)

**Comment by zkochan** [comment]
Sounds good.

**Comment by JoshuaKGoldberg** [comment]
Closing out my old issues for repos I no longer have context on. If anybody has a need for what this issue was asking about, I'd encourage them/you to file a new issue. Cheers!
